package.json

```diff
@@ -37,7 +37,7 @@
     "@typescript-eslint/eslint-plugin": "^2.0.0",
     "@typescript-eslint/parser": "^1.12.0",
     "fastify": "^2.10.0",
-    "fastify-plugin": "^1.5.0",
+    "fastify-plugin": "https://github.com/davidmarkclements/fastify-plugin",
```

Comment:
todo: update when/if https://github.com/fastify/fastify-plugin/pull/82 lands

Reply:
Can you restore this one?